dkist/io/file_manager.py

```diff
@@ -43,6 +43,12 @@ class DKISTFileManager:
     """
     __slots__ = ["_fm", "_inventory_cache", "_ndcube"]
 
+    def __str__(self) -> str:
+        return f"DKISTFileManager containing {len(self)} files with each array having shape {self.shape}"
```

Comment:
I'd definitely put basepath in this as it's the thing the user is most likely to change / care about.

I'm not sure if the array shape or the dimensionality of the files array is the most interesting thing either. 🤔

Reply:
you are phrasing this as "the array in each FITS file is this shape" is that what it is or is it "the array of filenames is this shape"?

Reply:
The latter. I think both are useful pieces of information and to be honest I'm not sure what other information you're looking for if you're printing the filemanager, other than basepath.

Reply:
Oh actually it's the latter as it stands, but that's not what I intended it to be I think :thinking: